test/e2e/ps_test.go

```diff
@@ -172,6 +172,24 @@ var _ = Describe("Podman ps", func() {
 		Expect(fullCid).To(Equal(result.OutputToStringArray()[0]))
 	})
 
+	It("podman ps --filter network=container:<idorname>", func() {
+		ctrAlpha := "alpha"
+		container := podmanTest.Podman([]string{"run", "-dt", "--name", ctrAlpha, ALPINE, "top"})
+		container.WaitWithDefaultTimeout()
+		Expect(container).Should(Exit(0))
+
+		ctrBravo := "bravo"
+		containerBravo := podmanTest.Podman([]string{"run", "-dt", "--network", "container:alpha", "--name", ctrBravo, ALPINE, "top"})
+		containerBravo.WaitWithDefaultTimeout()
+		Expect(containerBravo).Should(Exit(0))
+
+		result := podmanTest.Podman([]string{"ps", "-a", "--format", "table {{.Names}}", "--filter", "network=container:alpha"})
```

Comment:
Can you add another test with `container:<ID>` instead of name.

Reply:
Resolved in latest commit.